### ISSUE TITLE
Pretrained Models

### ISSUE THREAD

**Issue by AnanS** [sbos/AdaGram.jl#11]
First of all, great paper, thanks for making the source code available!

In the paper, it says that 'all trained models are available at http://github.com/sbos/AdaGram.jl'. However, I could not find any pretrained model on GitHub, and the project homepage (http://bayesgroup.ru/adagram) gives a 404.

I was wondering if the pretrained models are indeed available somewhere already, or if you have any plans of releasing them any time soon.

Thanks!

-- AnanS

**Comment by sbos** [comment]
Thank you for the interest in this project. We are planning to share the models soon, they are currently online in AWS S3 and if you contact with me by email I can send you links to them. 
Any suggestions of a proper free-traffic file hosting is welcome.

**Comment by alexanderpanchenko** [comment]
Here are couple of models available:

http://panchenko.me/data/joint/adagram/

these are trained on 60GB text corpus or wikipedia.

**Comment by ClaudeCoulombe** [comment]
Is it possible to get an idea of the parameters that have been used to train the pre-trained models (only those that are not the default setting)?

For example, DIM = 300, ALPHA = 0.2, MIN-FREQ = 20, SENSE-THRESHOLD = 1e-17

**Comment by alexanderpanchenko** [comment]
all params are defaults except alpha

**Comment by ClaudeCoulombe** [comment]
Nice, thank you Alexander!

**Comment by alexanderpanchenko** [comment]
You are welcome, you can cite this publication where the pretrained models
are described
http://www.lrec-conf.org/proceedings/lrec2016/pdf/625_Paper.pdf

**Comment by rahuldeve** [comment]
Hi Alexander,

Thank you for providing the pre-trained models. I was able to work with them a few weeks ago, but today the link seems to be down.

**Comment by alexanderpanchenko** [comment]
Some server maintenance...

**Comment by alexanderpanchenko** [comment]
It should be available soon. If you use these pretrained models please also cite this work:

http://www.lrec-conf.org/proceedings/lrec2016/summaries/625.html
